entity-framework/core/providers/index.md

```diff
@@ -46,18 +47,10 @@ We have been developing an EF Core provider for the SQL API in Cosmos DB.
 This will be the first complete document-oriented database provider we have produced, and the learnings from this exercise are going to inform improvements in the design of future releases of EF Core and possibly other non-relational providers.
 A preview is available on the [NuGet Gallery](https://www.nuget.org/packages/Microsoft.EntityFrameworkCore.Cosmos).
 
-### Oracle
-The Oracle .NET team has announced they are planning to release a first-party provider for EF Core approximately in the third quarter of 2018. See their [statement of direction for .NET Core and Entity Framework Core](http://www.oracle.com/technetwork/topics/dotnet/tech-info/odpnet-dotnet-ef-core-sod-4395108.pdf) for more information.
+### Oracle first-party provider
+The Oracle .NET team has published the beta of the [Oracle provider for EF Core](https://www.nuget.org/packages/Oracle.EntityFrameworkCore/).
 Please direct any questions about this provider, including the release timeline, to the [Oracle Community Site](https://community.oracle.com/).
 
-In the meanwhile, the EF team has produced a [sample EF Core provider for Oracle databases](https://github.com/aspnet/EntityFrameworkCore/tree/master/samples/OracleProvider).
```

Comment:
@ajcvickers I am removing this. Since the Oracle sample provider is no longer built in master, I don't know how much value there is in pushing it as an alternative to start your own provider.

Reply:
Sounds good.